XamlStyler.Console/XamlStylerConsole.cs

```diff
@@ -0,0 +1,251 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+using Xavalon.XamlStyler.Core;
+using Xavalon.XamlStyler.Core.Options;
+
+namespace Xavalon.XamlStyler.Xmagic
+{
+    public sealed class XamlStylerConsole
+    {
+        private readonly Options options;
+        private readonly StylerService stylerService;
+
+        public XamlStylerConsole(Options options)
+        {
+            this.options = options;
+
+            IStylerOptions stylerOptions = new StylerOptions();
+
+            if (this.options.Configuration != null)
+            {
+                stylerOptions = this.LoadConfiguration(this.options.Configuration);
+            }
+
+            ApplyOptionOverrides(options, stylerOptions);
+
+            this.stylerService = new StylerService(stylerOptions);
+        }
+
+        private void ApplyOptionOverrides(Options options, IStylerOptions stylerOptions)
+        {
+            if (options.IndentSize != null)
+                stylerOptions.IndentSize = options.IndentSize.Value;
+
+            if (options.IndentWithTabs != null)
+                stylerOptions.IndentWithTabs = options.IndentWithTabs.Value;
+
+            if (options.AttributesTolerance != null)
+                stylerOptions.AttributesTolerance = options.AttributesTolerance.Value;
+
+            if (options.KeepFirstAttributeOnSameLine != null)
+                stylerOptions.KeepFirstAttributeOnSameLine = options.KeepFirstAttributeOnSameLine.Value;
+
+            if (options.MaxAttributeCharactersPerLine != null)
+                stylerOptions.MaxAttributeCharactersPerLine = options.MaxAttributeCharactersPerLine.Value;
+
+            if (options.MaxAttributesPerLine != null)
+                stylerOptions.MaxAttributesPerLine = options.MaxAttributesPerLine.Value;
+
+            if (options.NoNewLineElements != null)
+                stylerOptions.NoNewLineElements = options.NoNewLineElements;
```

Comment:
For the non-nullable types, add brackets {}

Reply:
Not sure what you mean, they're all nullable :)
Do you want brackets on the `if` body?

Reply:
> Do you want brackets on the if body?

Yep